src/Dfe.ContentSupport.Web/Views/Shared/_VerticalNavigation.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@model List<PageLink>
```

Comment:
```suggestion
﻿@model IEnumerable<PageLink>
```

IMO changing this to an `IEnumerable` would be better suited to handle any future changes, as you're not currently using any of the `List` functionality, only the iterator.